style.css

```diff
@@ -67,11 +75,183 @@ li {
   align-items: center;
   text-align: center;
   color: #6070ff;
+  width: 100%;
+  gap: 70%;
+  background-color: #fff;
+  margin-bottom: -5px;
+  justify-content: center;
+}
+
+a {
+  text-decoration: none;
+}
+
+.middle-section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin: 0 auto;
+  gap: 50%;
   width: 90%;
+  border-radius: 12px;
+  box-sizing: border-box;
+  border: 1px solid #dfe1e6;
+  padding: 14px;
+  margin-top: 30%;
+  background: #fff;
+}
```

Comment:
- [ ] As demanded by the project requirements, kindly use CSS grid to layout the contents of the work section.